src/shipit_static_analysis/default.nix

```diff
@@ -4,7 +4,7 @@
 let
 
   inherit (releng_pkgs.lib) mkTaskclusterHook mkTaskclusterMergeEnv mkPython fromRequirementsFile filterSource ;
-  inherit (releng_pkgs.pkgs) writeScript gcc cacert gcc-unwrapped glibc glibcLocales xorg patch nodejs git python27 python35 coreutils shellcheck clang_5 zlib;
+  inherit (releng_pkgs.pkgs) writeScript gcc cacert gcc-unwrapped glibc glibcLocales xorg patch nodejs-8_x git python27 python35 coreutils shellcheck clang_5 zlib;
```

Comment:
maybe it would make sense to create a new assigment where you assign `nodejs-8_x` to `nodejs`.
then you:
   (1) wouldn't have change every mention of `nodejs`
   (2) also later updates become much easier and maybe less possibility to make mistakes.

above is just a personal taste so nothing you really have to fix, but maybe a good suggestion.